packages/thirdweb/.size-limit.json

```diff
@@ -2,18 +2,19 @@
   {
     "name": "thirdweb (esm)",
     "path": "./dist/esm/exports/thirdweb.js",
-    "limit": "45 kB",
+    "limit": "50 kB",
     "import": "*"
   },
   {
     "name": "thirdweb (cjs)",
     "path": "./dist/cjs/exports/thirdweb.js",
-    "limit": "110 kB"
+    "limit": "250 kB",
```

Comment:
why the massive bump? that should not be necessary?

Reply:
I'm not too sure if Webpack supports deep scope analysis for CJS. Maybe try import from `ox/Hex` (will also make aliasing look better).